fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1528,6 +1530,15 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
     @VariableMgr.VarAttr(name = SKIP_BAD_TABLET)
     public boolean skipBadTablet = false;
 
+    @VariableMgr.VarAttr(name = SKIP_RESOURCE_TAG_BAD_REPLICA, description = {
+            "当一个resource tag内的副本都不可查询的时候，默认情况下查询会失败。如果设置当前变量为true，"
+                    + "那么至少会找到一个可用的副本，保证查询的成功，这会导致查询路由到其他resource tag。",
+            "If none of the replicas within a resource tag are available for querying, the query will fail by default. "
```

Comment:
Do not use name   resource tag,  use resource group  in all documents and variable names and comments.